index.html

```diff
@@ -0,0 +1,214 @@
+          <h6>Tour</h6>
+          <p>
+            Features Fireworks and lazer shows, while the best music plays at
+            the background. A magical end to the festival this season.
+          </p>
+        </div>
+        <div class="act-three">
+          <img src="./images/tparty-50.png" alt="" />
+          <h6>Beach Party</h6>
+          <p>
+            Features Fireworks and lazer shows, while the best music plays at
+            the background. A magical end to the festival this season.
+          </p>
+        </div>
+        <div class="act-four">
+          <img src="./images/jump-50.png" alt="" />
+          <h6>Utopia Hours</h6>
+          <p>
+            Features Fireworks and lazer shows, while the best music plays at
+            the background. A magical end to the festival this season.
+          </p>
+        </div>
+        <div class="act-five">
+          <img src="./images/fireworks-50.png" alt="" />
+          <h6>Fireworks</h6>
+          <p>
+            Features Fireworks and lazer shows, while the best music plays at
+            the background. A magical end to the festival this season.
+          </p>
+        </div>
+      </div>
+      <div class="book-ticket">
+        <button class="ticket" id="bookticket" type="button">
+          Get Your Ticket
+        </button>
+      </div>
+    </section>
+    <section class="headliners" id="headliners">
+      <div class="artistss" id="artistss"></div>
+      <ul>
+        <li class="card1">
+          <img id="img1" src="." alt="cleanbandit image" />
+          <div class="awname">
+            <h4 id="ha"></h4>
+            <h5 id="h1"></h5>
+            <p id="p1"></p>
+          </div>
+        </li>
+        <li class="card2">
+          <img id="img2" src="." alt="David Guettta  image" />
+          <div class="dgname">
+            <h4 id="hb"></h4>
+            <h5 id="h2"></h5>
+            <p id="p2"></p>
+          </div>
+        </li>
+        <li class="card3">
+          <img id="img3" src="." alt="martin Garrix images" />
+          <div class="mname">
+            <h4 id="hc"></h4>
+            <h5 id="h3"></h5>
+            <p id="p3"></p>
+          </div>
+        </li>
+        <li class="card4">
+          <img id="img4" src="." alt="Avicii image" />
+          <div class="aname">
+            <h4 id="hd"></h4>
+            <h5 id="h4"></h5>
+            <p id="p4"></p>
+          </div>
+        </li>
+        <li class="card5">
+          <img id="img5" src="." alt="Dj Snake image" />
+          <div class="dname">
+            <h4 id="he"></h4>
+            <h5 id="h5"></h5>
+            <p id="p5"></p>
+          </div>
+        </li>
+        <li class="card6">
+          <img id="img6" src="." alt="Alan Walker" />
+          <div class="cname">
+            <h4 id="hf"></h4>
+            <h5 id="h6"></h5>
+            <p id="p6"></p>
+          </div>
+        </li>
+      </ul>
+    </section>
```

Comment:
- [ ] Please make sure this section is created dynamically without hard-coding and only loads when the page loads👀
- for example, you can have:  ```  <section class="headliners" id="headliners">
      <div class="artistss" id="artistss"></div>``` and the content are created dynamically like👇
 ```
{
  name: "Julia Leda", 
  role: "President of Young pirates of Europe", 
  description: "European ingetration, political democracy, and participation of ...", 
  photo: "image_url_here"
}
```

Reply:
Fixed!